src/design/singleProductionEnvelope.m

```diff
@@ -1,4 +1,4 @@
-function singleProductionEnvelope(model, deletions, product, biomassRxn, varargin)
+function [x, y1, y2] = singleProductionEnvelope(model, deletions, product, biomassRxn, varargin)
```

Comment:
The output I have was the one I could easily get from the function, to get everything might be more difficult since as we discussed not everything there is ordered and useful. I'll try when I write another test for this function with the suggestions you wrote below.